firefox-plugins-installer

```diff
@@ -43,11 +43,11 @@ JAVA_LINK=${PLUGINS_DIR}/libnpjp2.so
 # previous versions are often removed from the server, so it is safest
 # for an automated script to use the stable version in the archive
 # http://fpdownload.macromedia.com/get/flashplayer/pdc/11.2.202.643/install_flash_player_11_linux.x86_64.tar.gz
-FLASH_VERSION=11.2.202.644
-FLASH_SHA256SUM=aba0e2f5940fc2588357e93f7bad84a0db51792b2f2bd1d24be9ef491e87c041
-FLASH_FILENAME=fp_${FLASH_VERSION}_archive.zip
+FLASH_VERSION=24.0.0.221
```

Comment:
Correct, although it's kind of ugly to have that distinction, it would be nicer if we could have something more consistent, like using a prefix for the type, but I suppose something like this would not be too bad:
```
VERSION=<version>
URL=<url>
SHA256SUM=<sha256sum>

NPAPI_VERSION=<version>
NPAPI_URL=<url>
NPAPI_SHA256SUM=<sha256sum>
```
...even though I'd like it more if the first keys were prefixed by `PPAPI_` but, as you say, that would break current users so that's probably not a good idea, unless we also keep the former keys around for backwards compatibility for a while, as an upgrade path.

But that's probably too much of a hassle for something as simple as adding another possibility (and likely the only additional one we will ever need), so I think the unprefixed version for PPAPI plugins should be fine.